packaging/aznfs/DEBIAN/postinst

```diff
@@ -12,6 +12,61 @@ FLAG_FILE="/tmp/.update_in_progress_from_watchdog.flag"
 CONFIG_FILE="/opt/microsoft/aznfs/data/config"
 AUTO_UPDATE_AZNFS="false"
 
+install_stunnel()
+{
+    # Install stunnel package from source
```

Comment:
We can some kind of check for dependencies here to be safe (since I see make is part of Recommends and not Requires)-

    for cmd in wget tar make gcc; do
        command -v $cmd >/dev/null || { echo "$cmd is required but not installed. Exiting."; exit 1; }
    done

Reply:
wget should already be installed when it gets to postinst. gcc and make is also installed as requirements.